cmp/grid/GridModel.js

```diff
@@ -171,13 +167,15 @@ export class GridModel {
         enableExport = false,
         exportOptions = {},
         rowClassFn = null,
-        contextMenuFn = () => this.defaultContextMenu(),
```

Comment:
This was the extra closure layer that tripped me up.  I also thought this was a bit much to destructure here.